### PR TITLE
Updated Spotify API connection instructions

### PR DESCRIPTION
**Pull request by marcomicera**
I noticed that the `Create a Client ID` button doesn't exist anymore in the [Spotify dashboard](https://developer.spotify.com/dashboard/applications), so I updated the instructions accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -169,14 +169,17 @@ Instructions on how to set this up will be shown when you first run the app.
...
+    - You now can see your `Client ID` and `Client Secret`
 1. Now click `Edit Settings`
 1. Add `http://localhost:8888/callback` to the Redirect URIs
+1. Scroll down and click `Save`
 1. You are now ready to authenticate with Spotify!
 1. Go back to the terminal
 1. Run `spt`
 1. Enter your `Client ID`
 1. Enter your `Client Secret`
+1. Press enter to confirm the default port (8888)
```
Comment (Rigellute) [inline README.md:182]: This could be

```suggestion
1. Press enter to confirm the default port (8888) or enter a custom port
```

**Review by Rigellute — COMMENTED** [review]
Thank you for adding this @marcomicera, very helpful indeed! Just one minor comment on the port instruction (which I am grateful for you adding!).

**Comment by marcomicera** [comment]
> Just one minor comment on the port instruction (which I am grateful for you adding!).

Committed!

> Thank you for adding this @marcomicera, very helpful indeed! 

You're welcome! Thanks for creating this amazing tool.

**Review by Rigellute — APPROVED** [review]
Thank you once again @marcomicera 🙏

**Comment by Rigellute** [comment]
@all-contributors please add @marcomicera for doc

**Pull request merged**